zulipterminal/ui_tools/boxes.py

```diff
@@ -18,6 +18,8 @@ def __init__(self, view: Any) -> None:
         super(WriteBox, self).__init__(self.main_view(True))
         self.model = view.model
         self.view = view
+        self.is_edit_mode = False  # type: bool
```

Comment:
Maybe add an `in` to the variable name? *Maybe* remove the `is`?
(though see below)